compiler/record-minmax/src/RecordMinMax.cpp

```diff
@@ -32,6 +36,38 @@ using DataType = luci_interpreter::DataType;
 namespace
 {
 
+/**
+ * @brief  clipAndAverage does the following things
+ *         (1) Discard the top/bottom clip_ratio (%) values from vector
+ *             Ex: top_clip_ratio = 5, bottom_clip_ratio = 3
+ *                 discard top 5 % values and bottom 3 % values from vector
+ *         (2) Calculate the average of remaining values
+ */
+float clipAndAverage(std::vector<float> &vector, float top_clip_ratio, float bottom_clip_ratio)
+{
+  if (bottom_clip_ratio < 0 || bottom_clip_ratio >= 100)
+    throw std::runtime_error("Clip ratio (bottom) must be between 0 and 100");
```

Comment:
> It was intended. 

Message `0 and 100` does not correspond with your intention. I think it sould be like `between 0 and 99`

Reply:
Clip ratio is float, so I simply used 0 and 100 (99.1 is allowed).

Accurate expression would be "Clip ratio must be greater than or equal to 0 and less than 100.". I will change the message. Thanks.